lib/message/message.js

```diff
@@ -136,7 +142,7 @@ Message.fromJSON = function fromJSON (json) {
  */
 Message.prototype.toObject = function toObject () {
   return {
-    message: this.message
+    message: this.messageBuffer.toString()
```

Comment:
this part is not a good idea. the toObject should reliably some kind of object that you can actually use. probably you should put the hex or base64 version of the message here, not the string. because not all buffers can be represented with strings.

Reply:
actually, since this is an "object", it could just have the messageBuffer directly. in other words, just drop the .toString()

Reply:
...and also object.message should be object.messageBuffer

Reply:
Thanks! I've made some changes.

Please notice the test `roundtrip to-from-to'` changes and see if that is appropriate. We are now comparing the base64 representation with Message.toString()  (versus before the utf-8 string)